### DISCUSSION TITLE
How to turn on debug logging in Netbeans.

### DISCUSSION THREAD

**Discussion by warrenc5** [apache/netbeans#6376] [Q&A]
### Apache NetBeans version

Apache NetBeans 18

### What happened

How can I see DEBUG level logging messages and control the log file in Netbeans please.

I want to see what it's doing and understand why it's crashing all the time.

### How to reproduce

tail -f .netbeans/dev/var/log/messages 
shows INFO level only.

### Did this work correctly in an earlier version?

No / Don't know

### Operating System

Debian GNU/Linux 12

### JDK

openjdk 17.0.2 2022-01-18

### Apache NetBeans packaging

Own source build

### Are you willing to submit a pull request?

Yes

**Comment by lalo-mx** [comment]
There is a plugin that enables to change logging levels easily https://plugins.netbeans.apache.org/catalogue/?id=40 
![image](https://github.com/apache/netbeans/assets/7459881/20f38910-0804-4a17-bed9-aa60a77e24f1) 
![image](https://github.com/apache/netbeans/assets/7459881/08322d41-3f06-4ae8-8549-6ce6b445bfe5)
